src/stores/user.js

```diff
@@ -41,6 +41,25 @@ export const useUserStore = defineStore('user', {
     },
     signOut() {
       this.isSignedIn = false
+    },
+    async updateProfile(username) {
+      try {
+        const { data: { user } } = await supabase.auth.getUser()
+        const updates = {
+          id: user.id,
+          username,
+          updated_at: new Date()
```

Comment:
This should be a string or number (timestamp)

Reply:
And why is that? I mean, it doesn't seem to cause any issues?